packages/js-config-webpack-plugin/src/JsConfigWebpackPlugin.js

```diff
@@ -33,13 +33,25 @@ class JsConfigWebpackPlugin {
 	 * @returns {string}
 	 */
 	resolveBabelConfigFilePath(contextPath, environmentName) {
+		/**
+		 * Get the last result of an iterator
+		 * @param {Generator} iterator
+		 */
+		function getLast(iterator) {
+			let result = iterator.next();
+			while (!result.done) {
+				result = iterator.next();
+			}
+			return result.value;
+		}
+
 		// From https://github.com/babel/babel/blob/52a569056c6008c453bf26219461655c7d0322c4/packages/babel-core/src/config/files/package.js#L15
-		const packageData = findPackageData(contextPath);
+		const packageData = getLast(findPackageData(contextPath));
```

Comment:
What if checked if it is a generator function?

If it is we call it as generator function.
If it is not we call it as a normal function.

Found the code below with a quick search. There may be better ways to determine though.

https://github.com/tj/co/blob/717b043371ba057cb7a4a2a4e47120d598116ed7/index.js#L221

Reply:
Done :+1: